api/src/test/java/org/pmiops/workbench/TestLock.java

```diff
@@ -0,0 +1,24 @@
+package org.pmiops.workbench;
+
+public class TestLock {
```

Comment:
Should there be some kind of synchronization on this test lock?

Reply:
good catch